troposphere/static/js/components/modals/instance/launch/steps/ImageSelectStep.react.js

```diff
@@ -65,7 +132,7 @@ export default React.createClass({
     },
 
     renderMoreImagesButton: function (images, totalNumberOfImages) {
-        if (images.models.length < totalNumberOfImages) {
+        if (images.length < totalNumberOfImages) {
```

Comment:
I don't think it can here, because this is called within the render after an `if (images)` check on line 210.